ui/panels-plugin/src/plugins/time-series-chart/utils/data-transform.ts

```diff
@@ -29,7 +34,7 @@ import {
 
 export type RunningQueriesState = ReturnType<typeof useTimeSeriesQueries>;
 
-export const EMPTY_GRAPH_DATA = {
+export const EMPTY_GRAPH_DATA: EChartsDataFormat = {
```

Comment:
I ran into a type error with this at some point while working in here because this wasn't inferred to be the same as the other graph data. Explicitly typing it to match.